Library/Homebrew/livecheck.rb

```diff
@@ -13,12 +13,9 @@
 # [`brew livecheck` documentation](https://docs.brew.sh/Brew-Livecheck).
 class Livecheck
   extend Forwardable
+  extend T::Sig
```

Comment:
This can (and needs to be) removed due to https://github.com/Homebrew/brew/pull/15306